cockpit/README.md

```diff
@@ -0,0 +1,27 @@
+# VarkesUiApp
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 7.3.6.
```

Comment:
Who cares about that line, please write what the module is about and how to use it.